### PR TITLE
docs: Remove usages of `@see` tag, which is not TSDoc compliant

### PR DESCRIPTION
**Pull request by Josmithr**
Replaces usages of the `@see` tag in source-code comments. That tag is a part of the JSDoc spec, but not of TSDoc, which is what we use. Resulting documentation generation does not respect this tag, and ends up omitting referenced content.

This change replaces usages of that block with appropriate replacement syntax. 

Also includes a couple of related misc. fixes, and updates whitespace formatting for consistency.

### CODE REVIEW

**Review by tylerbutler — APPROVED** [review]
Some of the files from the PropertyDDS project are JS, just FYI in case you hadn't noticed that.

**Comment by Josmithr** [comment]
> Some of the files from the PropertyDDS project are JS, just FYI in case you hadn't noticed that.

Yeah. I debated whether or not to clean those up. In most cases, the syntax was wrong even for JSDoc. But since I don't think we're doing any sort of JSDoc rollup, and TSDoc is mostly a subset of JSDoc, I figured moving to TSDoc syntax was probably the right call.

File: common/lib/protocol-definitions/src/tokens.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export interface ITokenClaims {
      * Identifies the expiration time on or after which the token must not be accepted for processing.
      * Expressed in {@link https://en.wikipedia.org/wiki/Unix_time | Unix Time}.
      *
-     * @see https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4
+     * See https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4
```
Comment (ssimic2) [inline common/lib/protocol-definitions/src/tokens.ts:50]: Missing @link ?

**Review by ssimic2 — APPROVED** [review]

File: common/lib/protocol-definitions/src/tokens.ts (hunk 1 above)
Comment (Josmithr) [reply common/lib/protocol-definitions/src/tokens.ts:50]: Good catch! Will update.

**Pull request merged**